run_linux.go

```diff
@@ -165,9 +165,28 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	spec := g.Config
 	g = nil
 
-	logrus.Debugf("ensuring working directory %q exists", filepath.Join(mountPoint, spec.Process.Cwd))
-	if err = os.MkdirAll(filepath.Join(mountPoint, spec.Process.Cwd), 0755); err != nil && !os.IsExist(err) {
-		return err
+	wd := filepath.Join(mountPoint, spec.Process.Cwd)
+	logrus.Debugf("ensuring working directory %q exists", wd)
+	if err = os.MkdirAll(wd, 0755); err != nil {
```

Comment:
Maybe use `copier.Mkdir(mountPoint, wd, ...)` instead?  It attempts to chroot() to handle symbolic links safely.

Reply:
Done, PTAL

Reply:
Thanks!